Set secondary bonded interface as Managed (#2748)
theforeman/foreman-documentation: guides/common/modules/proc_adding-a-bonded-interface.adoc

Bonded interfaces use IDs in the form of *bond0* in the *Device Identifier* field.
+
A single *MAC address* is sufficient.
+

> If you are adding a secondary interface, you have to set the *Managed* flag as well.

Reviewer: ```suggestion
If you are adding a secondary interface, select *Managed*.
```
